runner/grpc.go

```diff
@@ -2,37 +2,100 @@ package runner
 
 import (
 	"context"
+
 	"github.com/compliance-framework/agent/runner/proto"
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-plugin"
+	"google.golang.org/grpc"
 )
 
+type AddHelper interface {
```

Comment:
Let's rename this to `ApiHelper` to make clear that is not adding a helper, but rather a helper to the API.